blob/singularity_store.go

```diff
@@ -0,0 +1,151 @@
+package blob
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/data-preservation-programs/singularity/client"
+	"github.com/data-preservation-programs/singularity/handler/dataset"
+	"github.com/data-preservation-programs/singularity/handler/datasource"
+)
+
+const MOTION_DATASET_NAME = "MOTION_DATASET"
+const MAX_CAR_SIZE = "31.5GiB"
```

Comment:
* Use camel case to stay consistent with general go naming conventions this repo follows?
* Don't export these values?